### ISSUE TITLE
Midpoint Review - Accessibility Feedback - IVC Forms, FMP 10-7959f-1 FMP Registration

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#77442]
## Next Steps for the VFS team

- [x] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [x] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Everything looks great, looking forward to seeing this go to code!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Engineering note:** In the accessibility annotations, you're indicating helper text at the top of each page that should be associated with the inputs via `aria-describedby` (example: [step 5](https://www.figma.com/file/PzB1F5TYuBK5KQgPbuhAwH/10-7959f-1?type=design&node-id=348-15182&mode=design&t=CEgAFJgsttyO2T4m-4), "Please include this information so that...").
   
   The design system web components don't actually make it easy to associate an input with text via `aria-describedby`! Since the `aria-describedby` would live on an `input` that's insulated inside the shadow DOM, it can't reliably reference any elements outside of the shadow DOM. (Screen reader support is split on crossing between regular DOM and shadow DOM.)
   
   The Design System Team came up with a workaround where components support a prop called `message-aria-describedby`, where you include the full text string of your helper text, rather than the ID of the text you're trying to reference. That means you end up with something like:
   
   ```
   <h3>Phone and email address</h3>
   <p>Please include this information so that we can contact you with questions or updates.</p>
   <va-text-input
      label="Phone number"
      message-aria-describedby="Please include this information so that we can contact you with questions or updates."
      name="phone number"
      onBlur={}
      onInput={}
      uswds
   />
   ```
   
   The `message-aria-describedby` text then populates a screen reader only `span` inside the shadow DOM that's then associated with the `input`. It's a workaround that works, but it's a bit duplicative. 
   
   An alternative approach that a lot of teams find works for them would be to use a `fieldset`/`legend` approach:
   
   ```
   <fieldset>
      <legend>
         <h3>Phone and email address</h3>
         <p>Please include this information so that we can contact you with questions or updates.</p>
      </legend>
      <va-text-input
         label="Phone number"
         name="phone number"
         onBlur={}
         onInput={}
         uswds
      />
   ```
   
   You can style the `p` inside the `legend` to look like regular text, but it gets announced as part of the `legend` when a screen reader user tabs into the first input. The net result is pretty much the same, but I feel like this is a little cleaner and less likely to somehow result in some kind of mismatch between the text screen reader users and non-screen reader users get. But both are valid!

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [x] Add the Collaboration Cycle initiative epic

**Comment by reiting** [comment]
Waiting for team review
